homeassistant/components/whirlpool/__init__.py

```diff
@@ -25,28 +26,29 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up Whirlpool Sixth Sense from a config entry."""
     hass.data.setdefault(DOMAIN, {})
 
+    session = async_get_clientsession(hass)
     region = CONF_REGIONS_MAP[entry.data.get(CONF_REGION, "EU")]
     brand = get_brand_for_region(region)
     backend_selector = BackendSelector(brand, region)
-    auth = Auth(backend_selector, entry.data[CONF_USERNAME], entry.data[CONF_PASSWORD])
+    auth = Auth(
+        backend_selector, entry.data[CONF_USERNAME], entry.data[CONF_PASSWORD], session
+    )
     try:
         await auth.do_auth(store=False)
-    except (aiohttp.ClientError, asyncio.TimeoutError) as ex:
+    except (ClientError, asyncio.TimeoutError) as ex:
```

Comment:
Any reason to change this? IMO, having the module name (aiohttp) keeps things bit more organized and I was under the impression that was the way most HA code used aiohttp. But I'm fine either way.

Reply:
In the import, I reduced to the used aiohttp classes - `ClientError` and `ClientSession`. This changed all previous `aiohttp.xxx`.  I thought we wanted to import the minimum as a "rule"